packages/v4/patternfly-docs/content/design-guidelines/components/pagination/pagination.md

```diff
@@ -4,15 +4,17 @@ section: components
 ---
 A **pagination** component gives users more navigational capability on pages with content views.
 
+## Usage 
 Add pagination above and below content views so that users can easily navigate as they read through listed items on a page.
 
 <img src="./img/full-page.png" alt="Example of pagination on full page table" />
 
 **1. Top pagination:** Always right-align pagination in the top toolbar, above content views.
```

Comment:
all the numbered lists should be indented but they're not - thinking it might be because the bolding is including the number, instead of only including the text if that makes sense